spec/factories/ads.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :ad do
+    address { "Somewhere far away" }
+    city { build(:city) }
```

Comment:
You can just put `city`, Factory bot will resolve it to this.

Reply:
Nice, didn't know that. Changed in https://github.com/vlado/earthquake-croatia/pull/131/commits/1a4584938ce9f8f1647cf4fc5345443824c91ad1